cloud/terraform/terraform_controller.py

```diff
@@ -32,12 +34,17 @@ def get_instances(self):
             for resource in resources:
                 if 'aws_instance' not in resource['address']:
                     continue
+
+                username = self.tf_configurator.get_username_by_instance_name(
```

Comment:
Yep, maybe it makes sense to use the "ami" key instead.

Reply:
That way we cannot have multiple instances from the same ami (we need this if we want to test instance type)

Reply:
Ooooh, that's true